### DISCUSSION TITLE
Front Matter: 2nd Cover & Declaration of originality for NOVA IMS MCSIG in Overleaf

### DISCUSSION THREAD

**Discussion by calliewentling** [joaomlourenco/novathesis#383] [Q&A] [answered]
Hello amazing nova thesis latex team, especially @joaomlourenco!  Thank you so kindly for automating this process.

I'm just about ready to submit my thesis but noticed I was missing some elements when I began adjusting the docstatus in Config/1_novathesis: Even when I have indicated "provisional" or "final", the second cover nor declaration of originality appear in my rendered PDF. I have tried adjusting some of the other declarations, yet the problem persists.

(I did search and see [Issue #234](https://github.com/joaomlourenco/novathesis/discussions/234) but it appears that this was then implemented into subsequent version of the documentation).

I´ve set my docdegree=msc, school=nova/ims, version 2022-05-16 in overleaf.

Is anyone else having such a challenge and how was it overcome?

Thank you very very much for the hard work! I'll be citing the template and recommending it to my peers!

Callie

**Comment by joaomlourenco** [comment, marked as answer]
@calliewentling 
Thank you for your enthusiasm with this template.  Please recommend it to your colleagues!  :)
Concerning your question, I just tested and it works perfectly for me (with msc+nova/ims+mcsig+provisional).  
Are you setting `\ntsetup{degreeacr/msc=mcsig}` in file `Config/9_nova_ims.tex`?

**Reply by calliewentling** [reply]
@joaomlourenco 
Absolutly I will recommend it!

Oh my goodness I finally realized my issue: in Config/4_files statements section I had uncommented the \ntaddfiles to try to force the inclusion of the statement. I only noticed when I was reviewing and transferring my content to a clean version! 😵 

Thank you so much for your prompt reply and guidance to resolve the issue. I think I'm finally ready to submit! 🥳 

All the best!

Callie

**Reply by joaomlourenco** [reply]
Good luck for the defense! 👍
